content/hardware/03.nano/boards/nano-esp32/tutorials/debugging/debugging.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Debugging with the Nano ESP32
+description: Get started with debugging using the Arduino Nano ESP32 with the IDE 2.
+author: Hannes Siebeneicher
+hardware:
+  - hardware/03.nano/boards/nano-esp32
+tags: [ESP32, Debugging, IDE]
+---
+
+Debugging is a crucial step especially when developing more complex projects. But even for beginners, it can be worth looking into as it gives you a lot more insights on what your code does at a specific time. 
```

Comment:
```suggestion
Being able to debug your board can be useful when developing more complex projects. It can prove useful for beginner or intermediate level projects as well, as it provides insight on what happens in your code does at a specific place. 
```